.pre-commit-config.yaml

```diff
@@ -74,7 +74,7 @@ repos:
         alias: black
         additional_dependencies: [black>=22.10.0]
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: "v1.15.0"
+    rev: "v1.16.0"
```

Comment:
It seems that when using this mirror, we're not having the proper type definitions for attr, which is leading to the exception:
```
mypy-python..............................................................Failed
- hook id: mypy
- exit code: 1

cosmos/operators/local.py:28: error: Module "attr" has no attribute "define"  [attr-defined]
Found 1 error in 1 file (checked 34 source files)
```

We should fix this before merging this PR.

Reply:
Fixed a fix in commit: 9d5b806f0de70ccf9f05a26b322965795c2db403 waiting for CI